Library/Homebrew/test/cmd/deps_spec.rb

```diff
@@ -18,4 +18,15 @@
       .and output("bar\nfoo\n").to_stdout
       .and not_to_output.to_stderr
   end
+
+  it "does not output dependencies that are already satisfied when --skip-installed is passed", :integration_test do
```

Comment:
Remove this test, sorry, we try to only have one per command because they are so slow.